openlibrary/templates/lists/widget.html

```diff
@@ -226,6 +205,26 @@ <h3>$('Add to List')</h3>
                 </a>
       </div>
     </div>
+      $if work_key and not readinglogonly:
+        <div class="my-reading-lists">
```

Comment:
Why does reading-lists become my-reading-lists?

Reply:
I was bringing this part out of .drop-down class and so I needed to modify some properties of this part. But reading-lists class is used in other elements. So I renamed the class of main div to "my-reading-lists".

Reply:
Where is the reading-lists class used? From what I can see, it's only used in one place - here.
```
jrobson@Jons-MBP openlibrary % ag reading-lists templates 
templates/lists/widget.html
186:                <div class="reading-lists">
208:                <div class="reading-lists">
```

Reply:
It's also used here in line number: 
208 187

Reply:
Okay. But if I want to edit this part specifically. Then the changes will be applied to other part which use "reading-lists" as well.

Reply:
Sure. Apologies for doing a bad thing of explaining myself.

So to clarify what I'm suggesting:
1): Let's move the existing styles in dropper.less into a new file called reading-lists.less. This file should have the top level selector .reading-lists
2) Make this HTML change and your required CSS changes to reading-lists.less
3) For the other usage of reading-list inside dropper, add some dropper specific styles e.g. `.dropper .reading-list`. These will override the default `.reading-list` styles in  the new reading-lists.less file.


Does this make sense?

Reply:
Got it. So I need to create a new .less file with .reading-list as main class. And this won't the other .reading-list because that's defined under .dropper class.
Thanks a lot.

Reply:
@jdlrobson I did as you said. I made a new .reading-lists file with styling for reading-lists class. I imported that in css/components/work.less. I am still getting error for max-size. 
![Screenshot from 2021-01-15 23-14-27](https://user-images.githubusercontent.com/62632865/104760571-91029880-5787-11eb-9405-3e64c8495ece.png)
This time for page-book.css [Possibly because work.less is imported in page-book.css]
Not sure about page-plain.css and page-edit.css and page-form.css?

Reply:
Don't worry about the bundlesize test for now.

Reply:
Can you add the new reading-lists file to the commit?

Reply:
Created pull request for reading-lists.less as well as work.less [Just imported reading-lists.less here]. I have done everything as instructed, but the change is not reflecting in the site and I can't find what I am missing.

The styling for reading-lists  in L185 is defined under dropper class in dropper.less.

Reply:
I can remove the readin-lists part from dropper class in dropper.less because currently I am using the same styling in reading-lists.less. But this may change in future.

Reply:
L185 should also change as part of this pull request .it appears to be the same interface but for  lists defined at an earlier date

Reply:
Yeah, it should change too. But the thing here is L185 is affected by two things:
- The reading-lists defined in dropper.less. [which can be removed, once reading-lists.less starts working]
- The reading-lists defined in reading-lists.less.
My major concern here is The reading-lists.less is not showing any effect in the site and I can't find what I am missing.

Reply:
The book page uses the CSS entry point `static/css/page-book.less`. THat includes the work component `static/css/components/work.less`. You've included the file there, so the new changes should show when you update the CSS using:
```
docker-compose exec web bash
make css
```

Reply:
I am getting this error`ERROR: No container found for web_1` while running `docker-compose exec web bash`.  I tried fully resetting the environment, but it didn't work.

Reply:
It's possible your container is `home` not `web`

Also make sure you run docker-compose up first.

Reply:
It's finally working. Thanks a lot.
![Screenshot from 2021-01-21 18-00-56](https://user-images.githubusercontent.com/62632865/105354175-31d4d600-5c16-11eb-91a1-fbb17a42d201.png)

Now should I remove the reading-lists section from dropper.less file? Because it has no other use apart from providing the formatting for reading-lists in L185 which can be done by reading-lists.less itself. 
Also the list in L185 appear as a dropdown when `work_key = None` in `widget.html`, do I need to make any change in it too?
![Screenshot from 2021-01-21 18-24-49](https://user-images.githubusercontent.com/62632865/105354301-5fba1a80-5c16-11eb-8c1c-fa5636a40fae.png)

Reply:
Glad to hear it!

Yes you can remove reading-list references inside dropper.

Regarding work_key:
Ideally these two should render exactly the same. I'll take a closer look tomorrow when i should have some spare time.

Reply:
Looking closely I believe the else clause applies to http://localhost:8080/subjects/science_fiction so you can test it there.

The menu can stay the way it is on the subjects page, but it would be better if all its CSS was inside reading-list component. If any CSS specific to that page is needed, feel free to add it to dropper.less